tests/candev/main.c

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright (C) 2019 Nalys
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the candev abstraction
+ *
+ * @author      Toon Stegen <tstegen@nalys-group.com>
+ * @author      Wouter Symons   <wsymons@nalys-group.com>
+ *
+ * @}
+ */
+
+#define ENABLE_DEBUG (0)
+
+#include <debug.h>
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <isrpipe.h>
+#include "shell.h"
+#include "can/device.h"
+
+#ifdef BOARD_NATIVE
+
+#include <candev_linux.h>
+
+candev_linux_t linux_dev;
+
+#else
+/* add other candev drivers here */
+#endif
+
+#define RX_RINGBUFFER_SIZE 128      //Needs to be a power of 2!
```

Comment:
```suggestion
#define RX_RINGBUFFER_SIZE 128      /* Needs to be a power of 2 */
```

Reply:
please apply throughout the code, i.e. replace `//` with `/* ... */` comments